crates/composition/tests/composition/objects_basic/supergraph.graphql

```diff
@@ -0,0 +1,6 @@
+type User {
```

Comment:
Do we expect some concrete order in the final type?

Reply:
As far as I can tell it's not specified by the spec and apollo produces stable but random order (probably the order it iterates the fields in different subgraphs). Not lexicographic order.